manuskript/models/abstractModel.py

```diff
@@ -104,9 +110,12 @@ def search(item):
         item = search(self.rootItem)
         return item
 
-    def getIndexByID(self, ID, column=0):
-        "Returns the index of item whose ID is `ID`. If none, returns QModelIndex()."
-        item = self.getItemByID(ID)
+    def getIndexByID(self, ID, column=0, ignore=None):
+        """Returns the index of item whose ID is `ID`. If none, returns QModelIndex().
+
+        It will ignore the given item as valid match for the ID."""
```

Comment:
I'd say "If `ignore` is set, will not return that item if found as valid match for the ID"

Reply:
I'll change it to that with a small change for better flow: "If `ignore` is set, **_it_** will not return that item if found as valid match for the ID."